functions-and-scope.js

```diff
@@ -12,7 +12,7 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe kan ik iedere waarde van de array checken op deze conditie?
 // * Hoe zorg ik ervoor dat dit ook werkt wanneer de array 100 entries bevat?
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan bijhouden?
-// Log het antwoord in de terminal.
+// Log het antwoord in de terminal
 
 // ---- Verwachte uitkomst: 6
 
```

Comment:
105: Je kan hier beter niet 0 maar de eerste waarde uit de array pakken. Het zal hier altijd goed gaan, omdat in de praktijk cijfers niet lager dan 0 kunnen zijn. Maar stel je voor dat je een array aanlevert van [-1, -2, -5, -3], dan krijg je 0 terug wat dus niet klopt.